selfdrive/car/ford/tests/test_ford.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python3
```

Comment:
you can rebase/merge master into your branch so you don't have these duplicate diffs on your PR